frontend/app/components/wp-edit/field-types/wp-edit-boolean-field.directive.html

```diff
@@ -1,10 +1,10 @@
 <input type="checkbox"
        class="wp-inline-edit--field wp-inline-edit--boolean-field"
        wp-edit-field-requirements="vm.field.schema"
-       ng-model="vm.workPackage[vm.fieldName]"
+       ng-model="vm.field.value"
        ng-false-value="false"
        ng-change="vm.handleUserSubmit()"
        ng-focus="vm.handleUserFocus()"
        ng-blur="vm.handleUserBlur()"
-       ng-disabled="vm.workPackage.inFlight"
+       ng-disabled="vm.field.inFlight"
```

Comment:
Yes, it's a visual feature of the _currently active_ field to disable editing it. We should keep that, in order to avoid changing the value currently being processed (e.g., in the path up until PATCHing the resource)